### PR TITLE
Server.log Match Results

### PR DESCRIPTION
**Pull request by AMZN-Gene**
-Print player results to server.log for tracking tournament winners if needed.
- Sort the players by score (highest score is 1st). If scores are matching, then sort by remaining armor.
image

### CODE REVIEW

File: Gem/Code/Source/Components/NetworkMatchComponent.cpp [hunk 1]
```diff
@@ -324,6 +324,8 @@ namespace MultiplayerSample
                 }
             }
 
+            // Print the player results to server.log for tracking tournament winners.
+            AZ_Info("NetworkMatchComponentController", "Match Results: Player %s score %i, armor %i.", state.m_playerName.c_str(), state.m_score, state.m_remainingArmor);
```
Comment (mbalfour-amzn) [inline Gem/Code/Source/Components/NetworkMatchComponent.cpp:328]: Nit: %i -> %u, these are unsigned ints.

Comment (mbalfour-amzn) [reply Gem/Code/Source/Components/NetworkMatchComponent.cpp:328]: Suggestion: If you use AZLOG(category, ...) then the printing could be off by default, but could be enabled on the server with a command-line argument.

Comment (allisaurus) [inline Gem/Code/Source/Components/NetworkMatchComponent.cpp:328]: how are we mapping Play <int> to player name as displayed on the score board?

Comment (AMZN-Gene) [reply Gem/Code/Source/Components/NetworkMatchComponent.cpp:328]: The log prints the player's name by using PlayerIdentityComponent

Comment (AMZN-Gene) [reply Gem/Code/Source/Components/NetworkMatchComponent.cpp:328]: Oh, would be good to add that cli as a comment in ILogger.h. Dont suppose you know the cvar off-hand?

Comment (allisaurus) [reply Gem/Code/Source/Components/NetworkMatchComponent.cpp:328]: > Would love to see sample of logs in situ.

+1

**Review by lmbr-pip — APPROVED** [review]

File: Gem/Code/Source/Components/NetworkMatchComponent.cpp (hunk 1 above)
Comment (AMZN-Gene) [reply Gem/Code/Source/Components/NetworkMatchComponent.cpp]: image

**Review by mbalfour-amzn — APPROVED** [review]

File: Gem/Code/Source/Components/NetworkMatchComponent.cpp (hunk 1 above)
Comment (AMZN-Gene) [reply Gem/Code/Source/Components/NetworkMatchComponent.cpp]: Updated to pretty print with a nice format and sorted
image

**Comment by AMZN-Gene** [comment]
This PR says it contains 50 modified audio files that havent been DCO'd even though o3de git already has these files. Closing and reopening a fresh PR #297. Apologies to those who already reviewed this :c
See #297

**Pull request closed without merging**